src/qrad3/trace.c

```diff
@@ -84,8 +84,7 @@ Loads the node structure out of a .bsp file to be used for light occlusion
 void MakeTnodes (dmodel_t *bm)
 {
 	// 32 byte align the structs
-	tnodes = malloc( (numnodes+1) * sizeof(tnode_t));
-	tnodes = (tnode_t *)(((int)tnodes + 31)&~31);
+	tnodes = aligned_alloc(32,(numnodes+1) * sizeof(tnode_t));
```

Comment:
I was way overthinking this, but this should just be
```
tnodes = malloc( (numnodes+1) * sizeof(tnode_t));
tnodes = (tnode_t *)(((intptr_t)tnodes + 31)&~31);
```
but the radiosity engine seems to produce nothing at all - just a black, empty lightmap, and I'm not sure where to go from here so I'll close this PR